<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stop Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>All Stops</string>
    </property>
    <widget class="QTableWidget" name="stopsTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>461</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deletestopbtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="createStopgrpbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Create new stop</string>
    </property>
    <widget class="QPushButton" name="createStopbtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cstop_namelineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cestimatedlineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cLoclineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Estimated Reaching Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="updateStopbox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Update stop</string>
    </property>
    <widget class="QPushButton" name="updateStopbtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stopidLinedit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ustop_namelineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uLoclineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uestimatedlineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop_ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Estimated Reaching Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addStoptoRoute">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Add stop to a route</string>
    </property>
    <widget class="QPushButton" name="Assignbtn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
    </widget>
    <widget class="QComboBox" name="RouteIDcomboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="StopIDcomboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Route_ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop_ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>All route-stops assignments</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>671</width>
       <height>192</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Route ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Starting Point</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Destination</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>R-1002</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>gulshan</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>clifton</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>9:40 AM</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>Inactive</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>R-1059</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Malir</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>Korangi</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>1:05 PM</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>Active</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="deletebtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="stopsRoutetable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="dashboardBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
